resources/user-agents/browsers/safari/safari-4-0.json

```diff
@@ -34,7 +34,7 @@
           ]
         },
         {
-          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Version/#MAJORVER#.#MINORVER#* Safari/*",
+          "match": "Mozilla/5.0 (#PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/#MAJORVER#.#MINORVER#*Safari/*",
```

Comment:
The goal with these changes was to try and shrink the Safari patterns so that other patterns that are based on these didn't have to be too much longer to work properly (the biggest one is Opera, since it just uses `OPR` as the identifier).